grammars/ruby haml.cson

```diff
@@ -24,7 +24,7 @@
     'beginCaptures':
       '1':
         'name': 'punctuation.section.comment.haml'
-    'end': '^(?! *$|\\1 )'
+    'end': '\\n?$'
```

Comment:
would `\n$` work? no reason to make the end of line optional, since it'll always be there